pkg/csi/service/common/types.go

```diff
@@ -85,7 +85,7 @@ type CreateVolumeSpec struct {
 	// TODO: Move this StorageClassParams
 	AffineToHost            string
 	VolumeType              string
-	VsanDirectDatastoreURL  string // Datastore URL from vSan direct storage pool
+	VsanDatastoreURL        string // Datastore URL used by host local volumes
```

Comment:
/ / Datastore URL used by host local volumes (vSAN Direct / vSAN SNA)

Can you mention type of datastores in the comment.